### PR TITLE
Update kodi-game repos to Kodi Matrix

### PR DESCRIPTION
**Pull request by garbear**
## Description

This updates the generated build system files to build using Matrix.

Based on https://github.com/kodi-game/game.libretro/commit/ea6c8e20e554b4c6f0e9cb5304db4568d8d08ffc

## How has this been tested?

Ran `process_game_addons.py` in an Azure Pipeline and built Yabause. Results: https://github.com/kodi-game/game.libretro.yabause/commit/b027382cc6477fa49283be3336279ca5898b580a

### CODE REVIEW

**Review by lrusak — APPROVED** [review]

**Comment by lrusak** [comment]
looks good. I assume we want to stick with the `master` branch for all the add-ons and not use `Matrix` branch like the non game add-ons use?

File: templates/addon/Jenkinsfile.j2 [hunk 1]
```diff
@@ -1 +1 @@
-buildPlugin(version: "Leia", platforms: {{ ['android-armv7', 'android-aarch64', 'ios-armv7', 'ios-aarch64', 'osx-x86_64', 'ubuntu-ppa', 'windows-i686', 'windows-x86_64']|reject("in", libretro_repo.exclude_platforms)|list()|tojson() }})
+buildPlugin(version: "Matrix", platforms: {{ ['android-armv7', 'android-aarch64', 'ios-armv7', 'ios-aarch64', 'osx-x86_64', 'ubuntu-ppa', 'windows-i686', 'windows-x86_64']|reject("in", libretro_repo.exclude_platforms)|list()|tojson() }})
```
Comment (lrusak) [inline templates/addon/Jenkinsfile.j2:1]: `ios-armv7` doesn't exist for Kodi master anymore. Also, `tvos-aarch64` should be added

Comment (lrusak) [reply templates/addon/Jenkinsfile.j2:1]: Do we even need to specify the platforms at all?

**Review by lrusak — CHANGES_REQUESTED** [review]
Jenkinsfile platforms need to be sorted out

File: templates/addon/Jenkinsfile.j2 (hunk 1 above)
Comment (garbear) [reply templates/addon/Jenkinsfile.j2:1]: I modelled this on https://github.com/kodi-game/game.libretro/commit/ea6c8e20e554b4c6f0e9cb5304db4568d8d08ffc, and it looks like I didn't fully update this file. Should be simpler now without the platforms.

**Comment by garbear** [comment]
>  I assume we want to stick with the master branch for all the add-ons and not use Matrix branch like the non game add-ons use?

Yes, game add-ons use the libretro API still at v1 and doesn't depend on kodi version.

**Comment by garbear** [comment]
@lrusak good to go?

**Review by lrusak — APPROVED** [review]

**Pull request merged**